src/WebJobs.Extensions.CosmosDB/Config/CosmosDBExtensionConfigProvider.cs

```diff
@@ -20,42 +21,25 @@ namespace Microsoft.Azure.WebJobs.Extensions.CosmosDB
     /// <summary>
     /// Defines the configuration options for the CosmosDB binding.
     /// </summary>
-    public class CosmosDBConfiguration : IExtensionConfigProvider
```

Comment:
We still need these types to be public, at least for now, as the build tools still probe for them. I know we talked about making all of these consistent and internal, but unfortunately, we can’t do that yet

Reply:
What probes for it? I made it internal in my changes as well. Otherwise we need to make several other internal service interfaces public.

Reply:
I think we should make them internal like they should be and update the tooling to work properly.

Reply:
I agree that it would be nice to change this, but if we do that now, we won't be able to use/test any of the extensions using the existing tooling until that is resolved.